sdcm/sct_config.py

```diff
@@ -1484,6 +1484,9 @@ class SCTConfiguration(dict):
         dict(name="use_dns_names", env="SCT_USE_DNS_NAMES", type=boolean,
              help="""Use dns names instead of ip addresses for nodes in cluster"""),
 
+        dict(name="enable_quota", env="SCT_ENABLE_QUOTA", type=bool,
+             help="Enable possibility of quota configuration on the cluster during setup")
```

Comment:
why it is required to enable like this?

Reply:
Actually we can enable quota on node during the nemesis and restart the node.
@fgelcer asked me to make it this way, so when it is enabled on the whole cluster we can use the nemesis or `configure_quota_on_node_for_scylla_user` separately.

Reply:
All this is not needed.